### PR TITLE
Feature/#413 time picker does not accept inline editing

### PR DESCRIPTION
**Pull request by manugallegob**

### CODE REVIEW

File: src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.business.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export const splitCSVContent = (csvContent: string): string[] => {
+  const splitedCsvContent = csvContent
+    .trim()
+    .split(/[:|,]/)
+    .map(el => el.trim());
+  return splitedCsvContent;
+};
+
+export const setTime = (csvData: string[]) => {
+  let [hora, minutos] = csvData;
```
Comment (brauliodiez) [inline src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.business.ts:10]: Can you name the varables in english ;)

File: src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.business.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  }
+  if (csvData[0] !== 'hh' || csvData[1] !== 'mm') {
+    if (
+      csvData.length > 2 ||
+      hora.length !== 2 ||
+      hora === '' ||
+      hora > '23' ||
+      minutos.length !== 2 ||
+      minutos === '' ||
+      minutos > '59'
```
Comment (brauliodiez) [inline src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.business.ts:22]: Avoid magic numbers in code, creat on top of the file, something like:

```ts
const MAX_DIGITS = 2;
const MAX_HOURS= '23';
const MAX_MINUTES = '59'
```

File: src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.business.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    .map(el => el.trim());
+  return splitedCsvContent;
+};
+
+export const setTime = (csvData: string[]) => {
+  let [hora, minutos] = csvData;
+  if (csvData.length < 2) {
+    return true;
+  }
+  if (csvData[0] !== 'hh' || csvData[1] !== 'mm') {
```
Comment (brauliodiez) [inline src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.business.ts:14]: Same here

const HOUR_MASK = 'hh';
const MINUTES_MASK='mm';

File: src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      restrictedHeight * 0.35,
+      20
+    );
+    const labelFontSize = Math.min(restrictedHeight * 0.3, 12);
+
+    const clockIcon = new window.Image();
+    clockIcon.src = clockIconSrc;
+
+    return (
+      <Group {...commonGroupProps} {...shapeProps}>
```
Comment (brauliodiez) [inline src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.tsx:76]: @deletidev can you check if we have alreaday some const values to apply?

Comment (deletidev) [reply src/common/components/mock-components/front-components/timepickerinput/timepickerinput-shape.tsx:76]: On svg, the font family need to be change to `font-family="Comic Sans MS, Balsamiq Sans, cursive", `.
Here on all Text,  fontFamily={INPUT_SHAPE.DEFAULT_FONT_FAMILY}, you may need to update spacing, 
 strokeWidth={INPUT_SHAPE.DEFAULT_STROKE_WIDTH}, and it is posible, control the size like on the existing input:
Captura de pantalla 2024-10-25 a las 19 41 00

Take a look to the `input-shape.tsx`.

**Pull request merged**